unit-tests/src/test/java/org/eclipse/collections/impl/multimap/AbstractMutableMultimapTestCase.java

```diff
@@ -322,4 +328,22 @@ public void testToString()
                 "{One=[1], Two=[2]}".equals(multimap.toString())
                         || "{Two=[2], One=[1]}".equals(multimap.toString()));
     }
+
+    @Test
+    public void testGetIfAbsentPut()
+    {
+        ArrayList<String> tempArrayList1 = new ArrayList<>();
+        tempArrayList1.add("Uno");
+        tempArrayList1.add("Ichi");
+
+        ArrayList<String> tempArrayList2 = new ArrayList<>();
+        tempArrayList2.add("Dos");
+        tempArrayList2.add("Ni");
+
+        FastListMultimap<String, ArrayList<String>> multimap = new FastListMultimap<>();
```

Comment:
It's confusing to use an iterable type as the value in a multimap. Why not just `MutableMultimap<String, String>`?